src/settings.test.js

```diff
@@ -0,0 +1,19 @@
+const { getSettings } = require('./settings');
+
+describe('replacements', () => {
+  it('should return nothing with no settings set', () => {
+    expect(getSettings()).toEqual(undefined);
+    expect(getSettings({})).toEqual(undefined);
```

Comment:
```suggestion
    expect(getSettings()).toBeUndefined();
    expect(getSettings({})).toBeUndefined();
```
[dust] just a thought 🤷‍♀️